src/components/userContent/cardTemplate/card.js

```diff
@@ -0,0 +1,63 @@
+import './card.css';
+import card from './card.hbs';
+import Header from './cardHeader/header';
+import OrderItem from './orderItem/orderItem';
+import ShowMore from './showMore/showMore';
+
+
+export default class Card {
+  render(target, props) {
```

Comment:
Please split render function into smaller functions. Try to keep size of each function under 5-10 lines.

Reply:
Example:

```
render() {
   const cardContext = {
      orderPrice: countOrderPrice(props.orders),
      active: props.header.active,
      button: props.orders.length > 0 ? 'Редактировать' : 'Заказать',
      sumPrice: props.orders.length > 0,
   };
   ...
}
countOrderPrice(order) {
...
}

```